src/app/pages/home/components/mode.component.ts

```diff
@@ -58,6 +61,19 @@ class ModeComponent extends HTMLElement {
 			}
 		});
 	}
+
+	disabledSettings = (disabled: boolean): void => {
+		let elements = this.querySelectorAll(".sc-mode__setting");
+		if (disabled) {
+			elements.forEach((element) => {
+				element.querySelector('app-btn-setting').setAttribute('data-disabled', 'true');
+			});
+		} else {
+			elements.forEach((element) => {
+				element.querySelector('app-btn-setting').setAttribute('data-disabled', 'false');
+			});
+		}
```

Comment:
Je crois que tout ça peut être résumé en une ligne : 
```suggestion
element.querySelector('app-btn-setting').setAttribute('data-disabled', String(disabled));
```

Qu’en dis-tu ?